crates/crabslab/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![feature(macro_metavar_expr)]
```

Comment:
I'd like to stay away from unstable features, if possible. Especially since `rust-gpu`'s `rustc` lags a bit behind...